osu/map.py

```diff
@@ -52,9 +56,30 @@ def files(self):
 
     def zip(self):
         map_pack_file = MAP_PACKS_FOLDER.joinpath(f"{uuid.uuid4()}.zip")
-
+        print(map_pack_file)
         files = self.files()
         FileTools.zip_files(files, to_file=map_pack_file)
-        FileTools.delete_files(files)
 
         return map_pack_file
+
+    async def upload(self) -> str | None:
+
+        await DbxClient.init(DROPBOX_ACCESS_TOKEN)
+
+        if DbxClient.client is None:
+            raise ValueError("DbxClient is not connected")
```

Comment:
check the params for initialise dropbox and theres a retries variable.

you can assume this shouldnt fail i would say